pkg/server/utils.go

```diff
@@ -146,3 +146,8 @@ func spdkMockServerCommunicate(rpc spdk.JSONRPC, l net.Listener, toSend []string
 func ProtoClone[T proto.Message](protoStruct T) T {
 	return proto.Clone(protoStruct).(T)
 }
+
+// ResourceIDToVolumeName creates name of volume resource based on ID
+func ResourceIDToVolumeName(resourceID string) string {
+	return fmt.Sprintf("//storage.opiproject.org/volumes/%s", resourceID)
```

Comment:
it is ok for Now
but in the future we will change the miffle part of URL from `volumes` to something more specific per each layer... frontend vs backend vs middleend maybe... 
I have no idea to what yet, but we will get there